docs/digger/installation/verify-installation.asciidoc

```diff
@@ -0,0 +1,10 @@
+== Verify Installation
+
+To help verify and monitor the status of the Digger service, the Digger installer will also install https://www.nagios.org/[Nagios] and some checks to monitor:
+
+* The status of the Jenkins container (pod)
+* The connectivity between Jenkins and the Mac servers (if any)
+* The load stats of the Mac servers (if any)
+
+The installer will automatically check the Nagios status at the end of the run.
+You can also manually login to the Nagios dashboard and check the status.
```

Comment:
I think the Ansible installer should print out the login details for Nagios and Jenkins, including the URLs, username and password. 

I have created a JIRA issue to track this issue. This way we don't need to document the steps of how to do that.